packages/amplify_datastore/test/test_models/Post.dart

```diff
@@ -85,23 +87,25 @@ class Post extends Model {
     var buffer = new StringBuffer();
 
     buffer.write("Post {");
-    buffer.write("id=" + id + ", ");
-    buffer.write("title=" + title + ", ");
-    buffer.write("rating=" + rating?.toString() + ", ");
-    buffer.write("created=" + created?.toDateTimeIso8601String() + ", ");
-    buffer.write("blog=" + blog?.toString());
+    buffer.write("id=" + "$id" + ", ");
+    buffer.write("title=" + "$title" + ", ");
+    buffer.write(
+        "rating=" + (rating != null ? rating.toString() : "null") + ", ");
+    buffer.write(
+        "created=" + (created != null ? created!.format() : "null") + ", ");
+    buffer.write("blog=" + (blog != null ? blog.toString() : "null"));
```

Comment:
Since this test model (along with others above) isn't codegen'd, maybe we should clean up the string interpolation. Also should not need to explicitly convert `null` to `"null"` just to stringify it.
```suggestion
    buffer.write("id=$id, ");
    buffer.write("title=$title, ");
    buffer.write("rating=${rating?.toString()}, ");
    buffer.write("created=${created?.format()}, ");
    buffer.write("blog=${blog?.toString()}"));
```

Reply:
Hmm why would we change the models to be different from codegen models?  Seems strange to prettify and clean up a model that should be representative of what codegen is.